skypy/pipeline/tests/test_items.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+
+def test_item():
+    from skypy.pipeline import Pipeline
+    from skypy.pipeline._items import Item
+
+    # set up a mock pipeline
+    pipeline = Pipeline({})
+
+    # construct the base class
+    item = Item()
+
+    # check `infer` method
+    item.infer({})
+
+    # check `depend` method
+    deps = item.depend(pipeline)
+    assert isinstance(deps, list)
+
```

Comment:
```suggestion
    assert len(deps) == 0

```
A stronger check, since we know base Item has no dependencies i.e. an empty list?